src/pages/onboarding/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { NextPage } from 'next';
+import { useState } from 'react';
+
+import SingleContentLayout from '@src/components/common/SingleContentLayout';
+import ContentFinish from '@src/components/onboarding/ContentFinish';
+import ContentJob from '@src/components/onboarding/ContentJob';
+import ContentName from '@src/components/onboarding/ContentName';
+import ContentYear from '@src/components/onboarding/ContentYear';
+import OnboardingLayout from '@src/components/onboarding/OnboardingLayout';
+
+const LAST_STEP = 3;
+const initialState: Onboarding = {
+  name: null,
+  job: null,
+  year: null,
+};
+export interface Onboarding {
+  name: string | null;
+  job: string | null;
+  year: number | null;
+}
+
+export type OnboardingKey = keyof Onboarding;
+
+const Onboarding: NextPage = () => {
+  const [currentStep, setCurrentStep] = useState(1);
+  const [onboardingValue, setOnboardingValue] = useState<Onboarding>(initialState);
+
+  const handleChangeOnboarding = (key: OnboardingKey, newValue: string | number) => {
+    setOnboardingValue((prev) => ({
+      ...prev,
+      [key]: newValue,
+    }));
+
+    setCurrentStep((prev) => prev + 1);
+  };
+
+  return (
+    <SingleContentLayout>
+      <OnboardingLayout
+        currentStep={currentStep > 3 ? undefined : currentStep}
+        lastStep={LAST_STEP}
+        title={getTitle(currentStep)}
+      >
+        {getContent(currentStep, handleChangeOnboarding, onboardingValue)}
+      </OnboardingLayout>
+    </SingleContentLayout>
+  );
+};
+
+const getTitle = (currentStep: number) => {
+  if (currentStep === 1) return '1. 닉네임 설정';
+  if (currentStep === 2) return '2. 직군 설정';
+  if (currentStep === 3) return '3. 연차 설정';
+
+  return undefined;
+};
```

Comment:
```suggestion
const TITLES: { [key in number]: string } = {
  1: '1. 닉네임 설정',
  2: '2. 직군 설정',
  3: '3. 연차 설정',
};
```

이런걸로 바꿔볼 수도 있겠네요.
물론 지금도 충분히 좋습니다 👍🏼